app/code/Magento/Email/view/adminhtml/layout/adminhtml_email_template_preview.xml

```diff
@@ -9,7 +9,6 @@
     <head>
         <remove src="extjs/resources/css/ext-all.css"/>
         <remove src="extjs/resources/css/ytheme-magento.css"/>
-        <remove src="css/styles-old.css"/>
```

Comment:
Thank you for your contribution. I think we should add some css to resolve this issue instead of including the old styles. Could you please check it again ?